libs/protocol-client/src/lib/client/contribution.ts

```diff
@@ -16,6 +16,9 @@ import {
 import { GraphQLClient } from 'graphql-request';
 import { paginate } from '../utils';
 
+const REQUIRED_CONFIRMATIONS = 1;
```

Comment:
This can be removed, The default argument is 1. We can remove passing in an argument to wait